src/components/Navbar/Navbar.css

```diff
@@ -21,9 +21,44 @@
 .topnav a:hover {
     background-color: #ddd;
     color: black;
+    border-radius: 25px;
+    border: #ddd;
+    margin-top: 10px;
+    block-size: auto;
+    display: inline;
 }
 
 .topnav a.active {
     background-color: #4CAF50;
     color: white;
 }
+.slack{
+    display: inline-block;
+    vertical-align:middle;
+    flex: 1;
+    margin-top: 10px;
+    width: 90px;
+    height: 50px;
+    resizeMode: 'contain';
+}
+.youtube{
+  vertical-align:middle;
+    margin-top: 10px;
+    width: 70px;
+    height: 50px;
+    resizeMode: 'contain';
+}
+.github{
+    margin-top: 10px;
+    vertical-align:middle;
+    width: 50px;
+    height: 50px;
+    resizeMode: 'contain';
+}
+.twitter{
+    margin-top: 10px;
+    width: 50px;
+    vertical-align:middle;
```

Comment:
multiple styles are repeated between these classes,

`margin-top`, `vertical-align` etc. Please put this in a single class .